### PR TITLE
Add email templates api swagger

### PR DESCRIPTION
**Pull request by vihanga-liyanage**

### CODE REVIEW

File: en/docs/develop/restapis/email-template.yml [hunk 1]
```diff
@@ -0,0 +1,588 @@
...
+        '401':
+          $ref: '#/components/responses/Unauthorized'
+        '403':
+          $ref: '#/components/responses/Forbidden'
+        '404':
+          $ref: '#/components/responses/NotFound'
+        '500':
+          $ref: '#/components/responses/ServerError'
+servers:
+  - url: 'https://localhost:9443/t/{tenant-domain}/api/server/v1'
```
Comment (ayshsandu) [inline en/docs/develop/restapis/email-template.yml]: change tenant-domain to localhost when adding docs

File: en/docs/develop/restapis/email-template.yml [hunk 2]
```diff
@@ -0,0 +1,588 @@
...
+      schema:
+        type: integer
+        format: int32
+        minimum: 0
+    sortOrderQueryParam:
+      in: query
+      name: sortOrder
+      required: false
+      description: >-
+        Define the order how the retrieved records should be sorted. _<b>This
```
Comment (sherenewso2) [inline en/docs/develop/restapis/email-template.yml]: change to "Define the order **in which** the retrieved records..."

File: en/docs/develop/restapis/email-template.yml [hunk 3]
```diff
@@ -0,0 +1,588 @@
...
+          $ref: '#/components/responses/Forbidden'
+        '500':
+          $ref: '#/components/responses/ServerError'
+    post:
+      tags:
+        - Email Template Types
+      summary: Add a new email template type.
+      operationId: addEmailTemplateType
+      description: |
+        Adds a new email template type to the system. An email template type can have any number of email templates.
```
Comment (sherenewso2) [inline en/docs/develop/restapis/email-template.yml]: In some places the descriptions start with "Adds" or "Retrieves" and in some places it starts with the singular form "Add" or "Retrieve". Shall we change it to one of these in all places for consistency?

File: en/docs/develop/restapis/email-template.yml [hunk 4]
```diff
@@ -0,0 +1,588 @@
...
+          $ref: '#/components/responses/NotFound'
+        '500':
+          $ref: '#/components/responses/ServerError'
+    post:
+      tags:
+        - Email Templates
+      summary: Add a new email template to an existing email template type.
+      operationId: addEmailTemplate
+      description: |
+        Adds a new email template an existing email template type in the system.
```
Comment (sherenewso2) [inline en/docs/develop/restapis/email-template.yml]: change to "Adds a new email template **to** an existing..."

File: en/docs/develop/restapis/email-template.yml [hunk 5]
```diff
@@ -0,0 +1,588 @@
...
+        The locale of the new email template should not already exists in the
+        respective email template type.
```
Comment (sherenewso2) [inline en/docs/develop/restapis/email-template.yml]: This sentence is not very clear..does it mean that the locale is not specified in the email template type and should be included in the email template?

File: en/docs/develop/restapis/email-template.yml [hunk 6]
```diff
@@ -0,0 +1,588 @@
...
+          $ref: '#/components/responses/Forbidden'
+        '404':
+          $ref: '#/components/responses/NotFound'
+        '500':
+          $ref: '#/components/responses/ServerError'
+  '/email/template-types/{template-type-id}/templates':
+    get:
+      tags:
+        - Email Templates
+      summary: Retrieve the list of email templates in the template type id.
```
Comment (sherenewso2) [inline en/docs/develop/restapis/email-template.yml]: change "id" to "ID" in all instances

File: en/docs/develop/restapis/email-template.yml [hunk 7]
```diff
@@ -0,0 +1,588 @@
...
+        '500':
+          $ref: '#/components/responses/ServerError'
+  '/email/template-types/{template-type-id}/templates/{template-id}':
+    get:
+      tags:
+        - Email Templates
+      summary: Retrieves a single email template.
+      operationId: getEmailTemplate
+      description: |
+        Retrieves the email template that matches to the template-type-id and the template-id
```
Comment (sherenewso2) [inline en/docs/develop/restapis/email-template.yml]: this sentence is missing a fullstop at the end

**Pull request merged**